lib/router.js

```diff
@@ -7,6 +7,7 @@
 
 var debug = require('debug')('Routr:router');
 var pathToRegexp = require('path-to-regexp');
+var queryString = require('query-string');
```

Comment:
use `query-string` or `qs`?  These 2 modules serialize query params differently.  Maybe check what express uses, I think they use `qs`: https://github.com/expressjs/express/blob/master/package.json#L48

Maybe it is better to use one of them as default, but allow user to customize with their own `parse` and `stringify` methods, so they can switch to anything they want.